ksql-engine/src/test/java/io/confluent/ksql/testutils/KafkaEmbedded.java

```diff
@@ -60,52 +60,35 @@
   /**
    * Creates and starts an embedded Kafka broker.
    *
-   * @param config Broker configuration settings.  Used to modify, for example, on which port the
-   *               broker should listen to.  Note that you cannot change some settings such as
-   *               `log.dirs`, `port`.
+   * @param config Broker configuration settings.  Used to modify, for example, on what listeners
+   *               the broker should use.  Note that you cannot change some settings such as
+   *               `log.dirs`.
    */
-  public KafkaEmbedded(Properties config) throws IOException {
-    tmpFolder = new TemporaryFolder();
-    tmpFolder.create();
-    logDir = tmpFolder.newFolder();
-    effectiveConfig = effectiveConfigFrom(config);
-    boolean loggingEnabled = true;
-
-    KafkaConfig kafkaConfig = new KafkaConfig(effectiveConfig, loggingEnabled);
+  public KafkaEmbedded(final Properties config) throws IOException {
+    this.tmpFolder = new TemporaryFolder();
```

Comment:
maybe: `TestUtils.tempDirectory()`

Reply:
`TemporaryFolder` works better should we want to restart the broker, as it gives us control of when the folder is deleted.